CMakeLists.txt

```diff
@@ -52,7 +53,11 @@ set(CMAKE_ALLOW_LOOSE_LOOP_CONSTRUCTS true)
 # Search path for cmake modules
 SET(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
 
-# Retrieve git version hash
+# Set release name if not provided via env var
+if(NOT DEFINED GIT_TAG_RELEASENAME)
+    set(GIT_TAG_RELEASENAME "Blessed Sanctuary")
```

Comment:
can this and the project version be placed in a separate place so we can easily keep track of them, either at the top or a separate file?

Reply:
Sure, shouldn't matter where in the file this gets set. I just put it next to the getversion include as that's where it's actually used, but shouldn't matter. I'll move it up to the top.